core/pom.xml

```diff
@@ -33,6 +33,10 @@
   <name>Spark Project Core</name>
   <url>http://spark.apache.org/</url>
   <dependencies>
+    <dependency>
```

Comment:
Why does this require a direct dependency now? at best, runtime?

Reply:
it was in json4s 3.5.3 and removed. so now need to be direct.

Reply:
Got it, looks like we do actually use it in the code, so does need to be explicit and compile scope.